app/knights/knights_data.py

```diff
@@ -0,0 +1,87 @@
+KNIGHTS = \
```

Comment:
Do not use line break `\`, just place the `{` on the first line and write the dict starting from a new line as it was previously.